aikau/src/main/resources/alfresco/lists/AlfFilteredList.js

```diff
@@ -466,6 +466,8 @@ define(["dojo/_base/declare",
       setupFilteringTopics: function alfresco_lists_AlfFilteredList__setupFilteringTopics(filter) {
          if (filter && filter.config && filter.config.fieldId)
          {
+            // See AKU-1076 - ensure filteringTopics do not require explicit configuration
+            !!!this.filteringTopics && (this.filteringTopics = []);
```

Comment:
Three exclamation marks seems an error. Either one or two (presumably 1?).

Also, the fact that you had to surround the second expression with brackets suggests that it's not good to hide variable assignments in this way. Perhaps a simple `this.filteringTopics = this.filteringTopics || []` might be ok?